resources/views/dashboard/_monthReminder.blade.php

```diff
@@ -12,7 +12,7 @@
                     @if ($reminderOutbox->reminder->contact->is_partial)
 
                         @php($relatedRealContact = $reminderOutbox->reminder->contact->getRelatedRealContact())
-                        <a href="{{ route('people.show', $relatedRealContact) }}">{{ $relatedRealContact->getIncompleteName() }}</a>
+                        <a href="{{ route('people.show', $relatedRealContact) }}">{{ $relatedRealContact->getNameAttribute() }}</a>
```

Comment:
Using a [mutator](https://laravel.com/docs/5.8/eloquent-mutators#accessors-and-mutators) means that the method `getNameAttribute()` allows you to call `$relatedRealContact->name;` to get the name.
```suggestion
                        <a href="{{ route('people.show', $relatedRealContact) }}">{{ $relatedRealContact->name }}</a>
```